types/types.go

```diff
@@ -226,6 +226,10 @@ type Image interface {
 	// The Digest field is guaranteed to be provided; Size may be -1.
 	// WARNING: The list may contain duplicates, and they are semantically relevant.
 	LayerInfos() []BlobInfo
+	// LayerDiffIDs returns a list of DiffIDs (= digests of the UNCOMPRESSED versions) of layers referenced by this image,
+	// in order (the root layer first, and then successive layered layers), if available, or nil if not.
+	// WARNING: The list may contain duplicates, and they are semantically relevant.
+	LayerDiffIDs() ([]digest.Digest, error)
```

Comment:
what's the difference with `LayerInfos()` above? can we make this clear? e.g. maybe `LayerInfos` can contain COMPRESSED digest?